open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        Args:
</pre><p>If your function have return values, you should include it in the documentation as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
ok